__tests__/app.test.js

```diff
@@ -127,26 +127,34 @@ describe('GET requests', () => {
                 expect(body.articles).toBeSorted({ key: "created_at", descending: true })
             })
         });
-        test('200 - should accept a topic query and respond with ', () => {
+        test('200 - articles endpoint should accept a topic query, filtering by specified value', () => {
             return request(app)
-            .get("/api/articles")
+            .get("/api/articles?topic=mitch")
             .expect(200)
-            .then(({body}) => {
-                expect(body.articles).toHaveLength(13)
+            .then(({ body }) => {
+                expect(body.articles).toHaveLength(12)
                 body.articles.forEach((article) => {
+                    expect(article).toHaveProperty("topic", 'mitch')
                     expect(article).toHaveProperty("author", expect.any(String))
                     expect(article).toHaveProperty("title", expect.any(String))
                     expect(article).toHaveProperty("article_id", expect.any(Number))
                     expect(article).not.toHaveProperty("body")
-                    expect(article).toHaveProperty("topic", expect.any(String))
                     expect(article).toHaveProperty("created_at", expect.any(String))
                     expect(article).toHaveProperty("votes", expect.any(Number))
                     expect(article).toHaveProperty("article_img_url", expect.any(String))
                     expect(article).toHaveProperty("comment_count", expect.any(Number))
                 }) 
                 expect(body.articles).toBeSorted({ key: "created_at", descending: true })
             })
-        });
+        })
+        test('200 - topic query does not match any articles', () => {
+            return request(app)
+            .get("/api/articles?topic=trombone")
+            .expect(200)
+            .then(({ body }) => {
```

Comment:
what if it is a topic like topic=paper which exists but doesnt have any articles